docs/snippets/web-components/login-form-with-play-function.js.mdx

```diff
@@ -0,0 +1,44 @@
+```js
```

Comment:
This was marked `n/a` on the [core snippet audit table](https://www.notion.so/chromatic-ui/Storybook-core-framework-snippet-audit-eb7bbbcc8e9a480f81fb5eaacc4f5c49), but I wasn't sure why. It's true that play functions can run into problems when web components use the Shadow DOM, but I don't think we need to assume that will be the case and omit the example entirely.

Reply:
I'm good with it. I'll update the snippet database accordingly and remove the example altogether. Once and if [this](https://github.com/konnorrogers/shadow-dom-testing-library) is added into the testing library and we make the change on our end, we can revisit this and add it back as with other examples. Sounds good?

Reply:
I think you misunderstood my comment.

I'm suggesting to _add_ this web components snippet. Because, even with the Shadow DOM complications, it still can be useful for other cases.

Reply:
Sorry for the misunderstanding. We can add it, sure, and while we're at it, add the remaining ones if you're ok with it.